.github/CODEOWNERS

```diff
@@ -621,7 +621,7 @@
 # PRLabel: %Monitor - Query
 /sdk/monitor/Azure.Monitor.Query/                                  @nisha-bhatia @JoshLove-msft @pallavit @Azure/azure-sdk-write-monitor-data-plane
 
-# ServiceLabel: %Monitor - Query
+# ServiceLabel: %Monitor - Query                                   @ToddKingMSFT
```

Comment:
This needs to be with the "ServiceOwners" line on L626.   The challenge is that Todd does not have the necessary write permissions to our repository to be in CODEOWNERS, so the linter will reject him.    He'll need to join the azure-sdk-write team, as the onboarding guide calls out.